### DISCUSSION TITLE
problem with FTP access

### DISCUSSION THREAD

**Discussion by spooky** [pockethost/pockethost#270] [Q&A]
Hello, I've created a pockethost account and I can log in fine, but using the same credentials, I'm unable to access the app data via ftp (invalid user or password) - password was generated by password manager so int contains funky characters and as far as I can see, it's not possible to change it anywhere.

Does anyone know how to deal with this?

**Comment by benallfree** [comment]
@spooky I've not heard of this one, what FTP client are you using?

**Reply by spooky** [reply]
Tried filezilla, total commander and win explorer.

**Reply by benallfree** [reply]
@spooky Log out, then go to Log In -> Forgot your password

You should be able to reset it there.

**Reply by spooky** [reply]
that helped. thanks!

**Comment by benallfree** [comment]
#94 and #86 also discussed this. I wonder if there was a regression.
